src/middleware/channel.ts

```diff
@@ -0,0 +1,70 @@
+import { NextFunction, Request, Response } from "express";
+import { GENERIC_CHANNEL_NAME } from "../config";
+import { logger } from "../util/logger";
+
+enum ChannelGroupLabel {
+  GOLDBACH = "die Goldbach Media",
+  IP = "die IP Österreich",
+  ORF = "der ORF",
+  P7S1 = "ProSiebenSat.1PULS4",
+  SERVUS_TV = "ServusTV",
+}
+
+const channelIdToChannelGroupMap: Record<number, ChannelGroupLabel> = {
```

Comment:
Probably it would be more elegant and less error prone to query this from the channel config.